### ISSUE TITLE
Implement Linear Search

### ISSUE THREAD

**Issue by ayan-biswas0412** [ayan-biswas0412/AlgoDocs#19]
Implement linear search algorithm in various languages given below

- [ ] C++
- [ ] C
- [ ] Java
- [ ] JavaScript
- [ ] Golang
- [ ] Ruby

Edit the file https://github.com/ayan-biswas0412/AlgoDocs/blob/main/docs/searching/linear_search.md

**Comment by JASKAMAL22** [comment]
Hey @ayan-biswas0412, I would like to do it in C++ as a part of hacktoberfest. Kindly assign it to me :)

**Comment by ayan-biswas0412** [comment]
> Hey @ayan-biswas0412, I would like to do it in C++ as a part of hacktoberfest. Kindly assign it to me :)

yeah go ahead and make an PR

**Comment by suranaakshat02** [comment]
I would like to perform it in c as a part of Hactoberfest .Kindly accept .

**Comment by Aayush-Gurung** [comment]
Hey @ayan-biswas0412  i would like to implement it in java, kindly assign it to me

**Comment by ayan-biswas0412** [comment]
> Hey @ayan-biswas0412 i would like to implement it in java, kindly assign it to me

Yeah sure go ahead, your contribution will help this project to grow more.

**Comment by Aayush-Gurung** [comment]
hey @ayan-biswas0412  I would like to implement it in javascript, kindly assign it to me.

**Comment by susheelg** [comment]
> I will be implementing for golang. :)

I have given a PR you can verify and mark Go
